tools/genwqe_cksum.c

```diff
@@ -48,15 +58,16 @@ static const char *version = GENWQE_LIB_VERS_STRING;
 static void usage(const char *prog)
 {
 	printf("Usage: %s [-h] [-v, --verbose] [-C, --card <cardno>|RED]\n"
-	       "\t[-V, --version]\n"
-	       "\t[-X, --cpu <only run on this CPU number>]\n"
-	       "\t[-D, --debug <create extended debug data on failure>]\n"
-	       "\t[-G, --use-sglist use the scatter gather list support]\n"
-	       "\t[-c, --check-result] check result against the software\n"
-	       "\t[-s, --bufsize <bufsize/default is 4KiB>]\n"
-	       "\t[-a, --adler32] use adler32 instead of crc32\n"
-	       "\t[-i, --pgoffs_i <offs>] byte offset for input buffer\n"
-	       "\t[FILE]...\n"
+	       "\t-C, --card <cardno> use cardno for operation (default 0)\n"
+               "\t-A, --accel = GENWQE | CAPI. (CAPI only for ppc64le)\n"
+	       "\t-V, --version show Software Version\n"
+	       "\t-X, --cpu <only run on this CPU number>\n"
+	       "\t-D, --debug <create extended debug data on failure>\n"
+	       "\t-G, --use-sglist use the scatter gather list support\n"
+	       "\t-c, --check-result] check result against the software\n"
```

Comment:
Here you have still the bracket. I wonder if the automatic man-page generation tool help2man depended on the format with the brackets? So that preserving them makes some sense.